benchmark_db_terraform/modules/benchmark_db/service/run_benchmark.sh

```diff
@@ -0,0 +1,25 @@
+#!/bin/bash
+
+## Enable strict error handling, command tracing, and pipefail
+set -euxo pipefail
+
+## Configure s3cmd
+s3cmd --dump-config \
+    --host="$BENCHMARK_ENDPOINT" \
+    --host-bucket="%(bucket)s.$BENCHMARK_ENDPOINT" \
+    --access_key="$AWS_ACCESS_KEY_ID" \
+    --secret_key="$AWS_SECRET_ACCESS_KEY" \
+    --multipart-chunk-size-mb=4096 > ~/.s3cfg
+
+## Run actual benchmark
+ruby bench.rb --chain calibnet --tempdir ./snapshots --daily
+ruby bench.rb --chain mainnet --tempdir ./snapshots --daily
+
+## Upload benchmark result to s3 weekly file
+week_number=$(date +%W) # Week starting on Monday
+s3cmd get s3://"$BENCHMARK_BUCKET"/benchmark-results/weekly-results/weekly_result_"$week_number".csv /tmp/weekly_result_"$week_number".csv --force || 
+echo "Timestamp,Forest Version,Lotus Version,Chain,Forest Snapshot Import Time [sec],Forest Validation Time [tipsets/sec],Lotus Snapshot Import Time [sec],Lotus Validation Time [tipsets/sec]" > /tmp/weekly_result_"$week_number".csv
```

Comment:
As we discussed in Slack, I would like it if the format was something like:
```
timestamp, forest-version, lotus-version, chain, metric, forest-value, lotus-value
```
We will add more metrics in the future.

Reply:
fixed